centralised-logging.txt

```diff
@@ -14,6 +14,7 @@ First thing we will need to open the firewall on the machine to allow connection
 
  firewall-cmd --zone=public --add-port=4000/tcp --permanent
  firewall-cmd --zone=public --add-service=http --permanent
+ firewall-cmd --zone=public --add-port=5601/tcp --permanent #Optional (to use kibana directly)
```

Comment:
Could you make it more clear, that this is only required, if kibana is used directly?